Yoast/wordpress-seo
# Show dashes correctly in react snippet preview

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Use the SnippetEditor prop to add WordPress specific url parsing: whitespace to dashes.

## Test instructions

This PR can be tested by following these steps:

* Use `define( 'YOAST_FEATURE_SNIPPET_PREVIEW', true );` in `wp-config.php` to show the React SnippetPreview.
* Type a slug with some spaces and check if they are shown as dashes in the preview.

## Quality assurance

~~* [ ] I have added unittests to verify the code works as intended~~

Fixes #9589
